gatsby-node.js

```diff
@@ -1,23 +1,98 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
 const path = require(`path`);
+const fetch = require('node-fetch');
+const xmlParser = require('fast-xml-parser');
 
-const BLOG_POST_TEMPLATE_PATH = path.resolve(`src/templates/blog-post.tsx`);
-const CLIENT_TEMPLATE_PATH = path.resolve(`src/templates/client-details.tsx`);
+const BLOG_POST_TEMPLATE_PATH = path.resolve('src/templates/blog-post.tsx');
+const CLIENT_TEMPLATE_PATH = path.resolve('src/templates/client-details.tsx');
+const CAREER_TEMPLATE_PATH = path.resolve('src/templates/career.tsx');
+const CAREER_DETAILS_TEMPLATE_PATH = path.resolve(
+  'src/templates/career-details.tsx',
+);
 
-exports.createPages = async ({ actions, graphql, reporter }) => {
+const PERSONIO_JOBS_URL = 'https://satellytes.jobs.personio.de/xml';
+const PERSONIO_SHORT_DESCRIPTION_NAME = 'Kurzbeschreibung';
+
+exports.createPages = async (createPagesArgs) => {
+  await createCareerPages(createPagesArgs);
+  await createBlogPages(createPagesArgs);
+  // await createClientPages(createPagesArgs);
+};
+
+const createCareerPages = async ({ actions }) => {
```

Comment:
Question, no action:
I understand that code is usually just stacked in `gatsby-node` but I would strongly prefer to have indidividual files to export those functions. That's not only cleaner but also cleaner to maintain & read, isn't it ?

Something like this:
```
import {createCareerPages} from some-file
import {createBlogPages} from some-other-file

exports.createPages = async (createPagesArgs) => {
  await createCareerPages(createPagesArgs);
  await createBlogPages(createPagesArgs);
};
```

Reply:
Yep, want to make all Gatsby config files in Typescript and also put the `createPages` into own files like we do in other project. Was just the first draft to make this thing work